src/Tutor.Core/DomainModel/IUnitRepository.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+using Tutor.Core.DomainModel.Units;
+
+namespace Tutor.Core.DomainModel
+{
+    public interface IUnitRepository
+    {
+        Unit GetUnit(int id);
+        List<Unit> GetUnits();
+
+        UnitKnowledgeComponent GetKnowledgeComponent(int id);
```

Comment:
We should remember to rename this UnitKC to KC when we finalize the refactoring.